src/views/Home/Home.test.jsx

```diff
@@ -10,6 +13,22 @@ const user = {
   color: 'crimson',
 }
 
-test('Should render the user profile', () => {
+test('Should render the user profile', async () => {
+  render(
+    <MemoryRouter>
+      <Home user={user} />
+    </MemoryRouter>
+  )
+  // name
+  const profileName = await screen.findByRole('heading', {
+    name: /vonta/i
+  })
 
+  // motto
+  // interests
+  // avatar
+  // header img
+  // list of likes
+
```

Comment:
looks like you are on the right track! You just need to grab the other elements by text or role.  Hint: the list has a role of `list`.  You can evaluate whether these items are present on the page by `expect(element).toBeInTheDocument()`.  As far as validating the list goes, you can treat it like an array and evaluate it with its `length` property.